src/panoptes/pocs/focuser/astromechanics.py

```diff
@@ -1,39 +1,47 @@
+from panoptes.utils import error
+from panoptes.utils.serial.device import find_serial_port
 from panoptes.pocs.focuser.serial import AbstractSerialFocuser
 
 
 class Focuser(AbstractSerialFocuser):
     """
-    Focuser class for control of a Canon DSLR lens via an Astromechanics Engineering Canon EF/EF-S adapter.
+    Focuser class for control of a Canon DSLR lens via an Astromechanics
+    Engineering Canon EF/EF-S adapter.
 
     Args:
         name (str, optional): default 'Astromechanics Focuser'
         model (str, optional): default 'Canon EF/EF-S'
+        vendor_id (str, optional): idVendor of device, can be retrieved with lsusb -v from terminal.
+        product_id (str, optional): idProduct of device, can be retrieved with lsusb -v from terminal.
 
-    Additional positonal and keyword arguments are passed to the base class, AbstractSerialFocuser. See
-    that class' documentation for a complete list.
+    Additional positonal and keyword arguments are passed to the base class,
+    AbstractSerialFocuser. See that base class documentation for a complete list.
 
     Min/max commands do not exist for the astromechanics controller, as well as
     other commands to get serial numbers and library/hardware versions. However,
     as they are marked with the decorator @abstractmethod, we have to override them.
     """
 
-    def __init__(self,
-                 name='Astromechanics Focuser Controller',
-                 model='astromechanics',
-                 *args, **kwargs):
+    def __init__(self, name='Astromechanics Focuser', model='Canon EF-232', vendor_id=None,
+                 product_id=None, *args, **kwargs):
+        # Check if have device, raise error.NotFound if unable to find.
+        self._serial_number = find_serial_port(vendor_id, product_id)
+        self._vendor_id = vendor_id
+        self._product_id = product_id
+
         super().__init__(name=name, model=model, *args, **kwargs)
-        self.logger.debug('Initialising Astromechanics Lens Controller')
+        self.logger.debug(f'Initializing {name}')
 
```

Comment:
I noticed that the move to initial position is currently done in the Birger class rather than the abstract serial class or abstract focuser class. Maybe it should be implemented here or in the base class for consistency?